examples/node/select_json_each_row_with_progress.ts

```diff
@@ -0,0 +1,39 @@
+import { createClient } from '@clickhouse/client'
+import { isProgress } from '@clickhouse/client-common'
```

Comment:
how about renaming to `isProgressItem` / `isProgressRow` / etc?

Reply:
Probably `ProgressRow` for the type, and `isProgressRow` for the type guard, then.